### PR TITLE
Fix install_conda in setup-ubuntu.sh to consider the CPU architecture

### LINKED ISSUE #8453: `install_conda` in `setup_ubuntu.sh` should consider CPU_TARGET
### Bug description

[Expected behavior]

The method  `install_conda` in `setup_ubuntu.sh` should download binary according to  `CPU_TARGET`.

[actual behavior]
It always downloads binary for `x86_64`.

### System information

Velox System Info v0.0.2
Commit: 8ed27bfacc7315528c400fe7f8282eda479d8c53
CMake Version: 3.25.1
System: Darwin-23.2.0
Arch: arm64
C++ Compiler: /Library/Developer/CommandLineTools/usr/bin/c++
C++ Compiler Version: 15.0.0.15000100
C Compiler: /Library/Developer/CommandLineTools/usr/bin/cc
C Compiler Version: 15.0.0.15000100
CMake Prefix Path: /Library/Developer/CommandLineTools/SDKs/MacOSX14.2.sdk/usr;/opt/homebrew;/usr/local;/usr;/;/opt/homebrew/Cellar/cmake/3.25.1;/usr/local;/usr/X11R6;/usr/pkg;/opt;/sw;/opt/local
  

### Relevant logs

```bash
Please not that I'm working on macos and try to build an dev image using ubuntu.
```

### PR DESCRIPTION
**Pull request by acvictor**
The function install_conda in setup_ubuntu.sh always downloads the conda package corresponding to
the CPU architecture x86_64.
Fix the function to download the conda package based on the actual CPU architecture (x86_64 or aarch64).

Resolves #8453

### CODE REVIEW

File: scripts/setup-ubuntu.sh [hunk 1]
```diff
@@ -114,9 +114,19 @@ function install_fbthrift {
 
 function install_conda {
   mkdir -p conda && cd conda
-  wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+  ARCH=$(uname -m)
+  
+  if [ "$ARCH" == "x86_64" ]; then
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
```
Comment (majetideepak) [inline scripts/setup-ubuntu.sh:120]: Can we coalesce the `if` and `if-else` by using
 `wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-$(ARCH).sh`

Comment (acvictor) [reply scripts/setup-ubuntu.sh]: Merged the wget for x86_64 and aarch64.

**Review by majetideepak — APPROVED** [review]

**Comment by acvictor** [comment]
Thank you @majetideepak for the review! @mbasmanova can you please help with completing this?

**Comment by majetideepak** [comment]
@mbasmanova I added some description. Thanks.

**Comment by mbasmanova** [comment]
@majetideepak Thank you, Deepak.

**Review by kgpai — APPROVED** [review]

**Comment by kgpai** [comment]
@majetideepak @acvictor We are having some issues importing PRs today; we hope to fix this soon, please give a day or two to merge and land these PRs.

**Review by Yuhta — APPROVED** [review]

**Comment by mbasmanova** [comment]
@acvictor Would you rebase so we can merge?

**Comment by acvictor** [comment]
> @acvictor Would you rebase so we can merge?

Done!

**Pull request closed without merging**